main.go

```diff
@@ -95,7 +95,17 @@ type file struct {
 	mode os.FileMode
 }
 
-func walk(ch chan<- *file, start string) {
+func walk(ch chan<- *file, path string) {
+	matches, _ := filepath.Glob(path)
```

Comment:
Windows wildcards are handled by the program itself, so the program receives the original text instead of matching files.

Reply:
Ok, sounds reasonable. Although PowerShell does support expansion.
Either way, I wonder what the implications are for the cases were a file name contains meta/expansion characters recognized by `filepath.Match`. Currently, `filepath.Glob` does this check:

```go
// hasMeta reports whether path contains any of the magic characters
// recognized by Match.
func hasMeta(path string) bool {
	// TODO(niemeyer): Should other magic characters be added here?
	return strings.ContainsAny(path, "*?[")
}
```

What if a path is something like `./mydir[hello]` and it's passed verbatim in a shell which supports expansion like bash:

    addlicense ./mydir\[hello\]/*

It might have undesirable effect, triggering `filepath.Glob` to try and expand `./mydir[hello]`.

We could add separate files with `+build` tags for Windows and Linux, but even on Windows, how would that work in DOS vs PowerShell without a confusion what expands/doesn't and at which point the expansion is done? Because I'm pretty sure `filepath.Match` has different set of expansion rules compared to PowerShell.

Reply:
An alternative for Windows, when not in PowerShell, could be to have another separate little program called `expand` and then feed the output of the `expand` to `addlicense`. It is probably out of scope for `addlicense` repo though.

Reply:
@marcacohen maybe you have other ideas?

Reply:
PowerShell is also only partial command support, which is also command itself support, not PowerShell support.
So I don't think you have to worry about the difference DOS vs PowerShell.

Reply:
Passed verbatim in a shell which supports expansion, this is a problem.